carma_wm/include/carma_wm/CARMAWorldModel.h

```diff
@@ -160,6 +161,13 @@ class CARMAWorldModel : public WorldModel
    */
   lanelet::CarmaTrafficSignalPtr getTrafficSignal(const lanelet::Id& id) const;
 
+  /*! \brief update minimum end time to account for minute of the year
+   */
+  boost::posix_time::ptime min_end_time_converter_minute_of_year(boost::posix_time::ptime min_end_time,bool moy_exists,uint32_t moy);
```

Comment:
Please put more description for the inputs. what does moy stand for. 
Also maybe it's better to make default value of `moy = 0`  here since `moy_exists` can be false